### PR TITLE
Feature new sample worlds

### PR DESCRIPTION
**Pull request by fabienrohrer**
Address #719

- [x] Add an `accelerometer.wbt` example
- [x] Add an `brake.wbt` example
- [x] Add these worlds to the guided tour
- [x] Doc and screenshots
    - https://cyberbotics.com/doc/guide/samples-devices?version=feature-new-sample-worlds#accelerometer-wbt
    - https://cyberbotics.com/doc/guide/samples-devices?version=feature-new-sample-worlds#brake-wbt
- [x] Log
    - https://cyberbotics.com/doc/reference/changelog-r2019?version=feature-new-sample-worlds#webots-r2019b-revision-1

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Current result:

- accelerometer.wbt: accelerometer
- brake.wbt: brake

**Comment by fabienrohrer** [comment]
👍

**Comment by DavidMansolino** [comment]
:eyes:

File: projects/samples/devices/controllers/brake/brake.c [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  while (wb_robot_step(time_step) != -1) {
+    if (wb_robot_get_time() == 4.0) {
+      // At four seconds, the movement of the linear motors is started.
+      printf("Brake activated!\n");
+      wb_motor_set_position(linear_motor_a, wb_motor_get_max_position(linear_motor_a));
+      wb_motor_set_position(linear_motor_b, wb_motor_get_max_position(linear_motor_b));
+    }
+    if (wb_robot_get_time() > 4.13) {
+      // When the red blocks touch the wheel, the braking of the main engine is performed.
+      wb_brake_set_damping_constant(brake, 10.0);
```
Comment (DavidMansolino) [inline projects/samples/devices/controllers/brake/brake.c]:
```suggestion
      wb_brake_set_damping_constant(brake, 1.0);
```
Same comment here, IMO it is nicer to see the wheel decelerating than seeing the wheel stopping almost instantaneously.

**Comment by fabienrohrer** [comment]
@DavidMansolino Thank you, all your comments were pertinent and I think I addressed them all.

👍

**Pull request merged**